### ISSUE TITLE
Freeserf.net does not create user.cfg on launch.

### ISSUE THREAD

**Issue by GraionDilach** [Pyrdacor/freeserf.net#108]
On Windows 10.

This does not affect gameplay and/or asset lookups, but weird.

EDIT: It is created when the settings are updated, but the log file still reports it missing until something is changed from default.

**Comment by Pyrdacor** [comment]
The game looks for the config on startup. If not found it will use the internal default settings. But it should create it on game close imho. But if it only creates it for non-default settings it would be also fine as it will load the default settings anyway if the config is missing. I can have a look at it. What would be your desired/expected behavior?

**Comment by GraionDilach** [comment]
The expectation would be that it creates on first startup. Most of the programs I used did this - generating the config file imminently instead of warning about it missing. The unintuitive part is that it warns you for having it missing without even trying to create it. I started up the game with throwing in the DOS art and the Amiga music and I was unable to recognize which SFX version it loaded (the autodetector did fall back to PreferAmiga on the music, that was nice).
